src/rules/sortImportsRule.ts

```diff
@@ -0,0 +1,242 @@
+'use strict';
+
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+
+const RULE_NAME = 'sort-imports';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public static metadata: Lint.IRuleMetadata = {
+    ruleName: RULE_NAME,
+    description: 'enforce sorting import declarations within module',
+    descriptionDetails: Lint.Utils.dedent`
```

Comment:
Try to use some of the infomation provided by eslint to fill in the optionsDescription is the rest of the metadata. Also, if you add anything new that is not eslint standard please document it here. Here is the link to the eslint sort-imports: http://eslint.org/docs/rules/sort-imports